src/coreclr/jit/compiler.h

```diff
@@ -4092,16 +4098,7 @@ class Compiler
                                                 // that writes to arg0
 
 #ifdef TARGET_X86
-    bool lvaIsArgAccessedViaVarArgsCookie(unsigned lclNum)
-    {
-        if (!info.compIsVarArgs)
-        {
-            return false;
-        }
-
-        LclVarDsc* varDsc = lvaGetDesc(lclNum);
-        return varDsc->lvIsParam && !varDsc->lvIsRegArg && (lclNum != lvaVarargsHandleArg);
-    }
+    bool lvaIsArgAccessedViaVarArgsCookie(unsigned lclNum);
```

Comment:
By the way, there is another function that does exactly the same: `raIsVarargsStackArg`.

(I only noticed that after adding this one...)

Reply:
Ah, I didn't notice that. I'll delete that one.